lib/index.js

```diff
@@ -307,9 +308,29 @@ If you don't have a key, head over to https://binaris.com to request one`);
   (use "bn create node8 hello" to create a Node.js template function in your CWD)`);
 };
 
+/**
+ * Allow feedback by user of a given account.
+ *
+ * @param {object} options - Command line options.
+ */
+const feedbackHandler = async function feedbackHandler(options) {
+  if (!options.email || options.email.length === 0) {
+    throw new Error('Not a valid email.');
+  }
+  if (!options.message || options.message.length === 0) {
+    throw new Error('Not a valid message.');
+  }
+  const funcData = { email: options.email, message: options.message };
+  await feedback(funcData);
+  logger.info(
+    `Thank you! If you need help using Binaris please join our support slack channel at
+    https://binaris-community.slack.com.`);
```

Comment:
It appears we can create a link that expires either monthly or on a 1000-use basis. Ill see if there is a way we can generate this, since I think that has wider implications for fostering/managing our community.